source/renderer/app/themes/daedalus/cardano.js

```diff
@@ -307,6 +307,11 @@ export default {
 
   '--theme-terms-of-use-text-color': '#5e6066',
 
+  '--theme-data-migration-background-color': '#2C4567',
```

Comment:
@daniloprates please always use lowercase characters for CSS colors 🙏 
(it is the same situation in all 3 theme files)

Reply:
✅